plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -407,10 +424,99 @@ public synchronized Optional<ConnectorOutputMetadata> finishCreateTable(
         requireNonNull(info, "info is null");
 
         tables.put(tableName, new TableInfo(info.columns(), info.properties(), info.comment()));
+        createViewWithPredicates(tableName, info.columns(), computedStatistics);
 
         return Optional.empty();
     }
 
+    private synchronized void createViewWithPredicates(SchemaTableName tableName, List<ColumnInfo> columns, Collection<ComputedStatistics> computedStatistics)
```

Comment:
I used a view to avoid parsing the predicate values from strings. I actually like how we can generate the view and allow users to fine-tune it, without adding a bunch of separate column properties, that would be as hard to edit as the view.